testsuite/tests/ocaml_api/general/main.ml

```diff
@@ -217,6 +217,22 @@ let test_node () =
     (FooNode.for_all aux root) ;
   Format.printf "@[<v>============================@ @ @]"
 
+let test_lookup_with_kind () = 
+  Format.printf "@[<v>========== LOOKUP ==========@ @]" ;
+  let ctx = AnalysisContext.create () in
+  let u = AnalysisContext.get_from_file ~reparse:true ctx "foo.txt" in
+  print_exit_if_diags u ;
+  let root = root_exn u in
+  let lookup_1 = FooNode.lookup_with_kind Sequence root {Sloc.line=1; Sloc.column=25} in
+  (match lookup_1 with
```

Comment:
It seems that you did not run ocamlformat on the code. I would prefer you do so.
We changed ocamlformat version recently so other parts of the code will change with a call to ocamlformat unfortunately, but this is fine

Reply:
Done in  c076216